internal/pkg/app/app.go

```diff
@@ -54,12 +54,5 @@ func (a *App) Run(srcCustomGrafanaClient CustomGrafanaClient) error {
 	log.Info()
```

Comment:
Is this empty log message just to create a newline? You could just start the next one with '\n' , and then remove this. That way each call to log actually has a corresponding message.